### PR TITLE
Documentation/requirements

### PR DESCRIPTION
**Pull request by albertinisg**
Update documentation according the [developers guidelines](http://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/Developer_Guidelines).

### CODE REVIEW

**Comment by albertinisg** [comment]
@dmabtrg @jmcanterafonseca review?

File: README.md [hunk 1]
```diff
@@ -10,9 +10,18 @@
...
+  - [Installation and Administration Guide](#installation-and-administration-guide)
   - [Deploying on a Cloud Fiware Lab Instance](#deploying-on-a-cloud-fiware-lab-instance)
-- [REST Interface](#rest-interface)
+- [User and Programmers Manual](#user-and-programmers-manual)
+  - [Miscellaneous configuration](#miscellaneous-configuration)
+    - [How to configure a Context Provider in Orion](#how-to-configure-a-context-provider-in-orion)
+    - [How to configure Cygnus in TourGuide](#how-to-configure-cygnus-in-tourguide)
+    - [How to generate a new restaurant data image](#how-to-generate-a-new-restaurant-data-image)
+    - [How to run Feeders for the data image generation](#how-to-run-feeders-for-the-data-image-generation)
+    - [How to retrieve an Oauth Token to request the API](#how-to-retrieve-an-oauth-token-to-request-the-api)
```
Comment (jmcanterafonseca) [inline README.md]: How to retrieve an Oauth Token to use the API

File: README.md [hunk 2]
```diff
@@ -57,12 +68,49 @@ You can deploy an instance of the Fiware TourGuide App in a cloud instance using
...
+## User and Programmers Manual
+
+The complete user guide of this environment is available at [Readthedocs](http://fiwaretourguide.readthedocs.org/en/latest/fiware-tour-guide-application-a-tutorial-on-how-to-integrate-the-main-fiware-ges/fiware-tour-guide-application-a-tutorial-on-how-to-integrate-the-main-fiware-ges/).
+
+### Miscellaneous configuration
+
+#### How to configure a Context Provider in Orion
+
+[Here](https://github.com/Fiware/tutorials.TourGuide-App/blob/master/doc/ContextProvider.md) you can find a simple example of how to configure a Context Provider in TourGuide using Orion.
+
```
Comment (jmcanterafonseca) [inline README.md:80]: I think we don't need 'in TourGuide'

File: README.md [hunk 3]
```diff
@@ -57,12 +68,49 @@ You can deploy an instance of the Fiware TourGuide App in a cloud instance using
...
+#### How to generate a new restaurant data image
+
+The image provided is based on [Euskadi Open Data information](http://opendata.euskadi.eus/contenidos/ds_recursos_turisticos/restaurantes_sidrerias_bodegas/opendata/restaurantes.json). The image is already loaded, but we provide information on [how to generate it yourself](https://github.com/Fiware/tutorials.TourGuide-App/blob/master/docker/images/tutorials.tourguide-app.restaurant-data/Readme.md).
+
+#### How to run Feeders for the data image generation
+
+Find out how to load information into a new image using the [Tourguide feeders](https://github.com/Fiware/tutorials.TourGuide-App/tree/master/server/feeders).
+
+#### How to retrieve an Oauth Token to request the API
```
Comment (jmcanterafonseca) [inline README.md]: to call the API

File: README.md [hunk 4]
```diff
@@ -57,12 +68,49 @@ You can deploy an instance of the Fiware TourGuide App in a cloud instance using
...
+
+Every request against the TourGuide API must be authenticated. For that purpose, we provide [a simple script](https://github.com/Fiware/tutorials.TourGuide-App/tree/master/server/misc) to generate an Oauth Token based on a username and password of the [preloaded data](https://github.com/Fiware/tutorials.TourGuide-App/tree/master/docker/images/tutorials.tourguide-app#idm-users-organizations-apps-roles-and-permissions).
```
Comment (jmcanterafonseca) [inline README.md]: Every TourGuide API request

**Comment by jmcanterafonseca** [comment]
LGTM with the tiny comments addressed

**Comment by albertinisg** [comment]
@jmcanterafonseca done in eba2cf682e709bf2106550499a154c8cc06e6b44

**Comment by jmcanterafonseca** [comment]
LGTM thanks

**Comment by dmabtrg** [comment]
LGTM.

**Pull request merged**